packages/solid-query/src/createMutation.ts

```diff
@@ -95,8 +42,8 @@ export function createMutation<
   })
 
   createComputed(() => {
-    const newParsedOptions = parseMutationArgs(arg1, arg2, arg3)
-    setOptions(newParsedOptions)
+    const newParsedOptions = { ...mutationOptions }
+    setOptions(mutationOptions)
```

Comment:
is it necessary to create a shallow copy of `mutationOptions` ?

Reply:
I don't think so, I have just tried to remove it and the tests are passing .. will remove the shallow copy

Reply:
Done ✅